server/ai_process.go

```diff
@@ -33,6 +33,7 @@ const defaultUpscaleModelID = "stabilityai/stable-diffusion-x4-upscaler"
 const defaultAudioToTextModelID = "openai/whisper-large-v3"
 const defaultLLMModelID = "meta-llama/llama-3.1-8B-Instruct"
 const defaultSegmentAnything2ModelID = "facebook/sam2-hiera-large"
+const defaultLiveVideoToVideoModelID = "stream-diffusion" // TODO what should this be?
```

Comment:
I guess stream diffusion is not on huggingface, so you can leave it as `stream-diffusion` or change to `cumulo-autumn/stream-diffusion`

Reply:
Changed to `cumulo-autumn/streamdiffusion` in https://github.com/livepeer/go-livepeer/commit/9e461a7d2ee59b1cd5603382d403c082b71469ad